File: C/dardo.c
#include <stdio.h>

int main() {

    double d1, d2, d3, maior;

    printf("Digite as tres medidas:\n");
    scanf("%lf", &d1);
    scanf("%lf", &d2);
    scanf("%lf", &d3);

    if (d1 > d2 && d1 > d3) {
        maior = d1;
    }
    else if (d2 > d3) {
        maior = d2;
    }
    else {
        maior = d3;
    }

    printf("MAIOR DISTANCIA = %.2lf", maior);

    return 0;
}

File: Java/Dardo.java
import java.util.Locale;
import java.util.Scanner;

public class Dardo {
    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        double medida1, medida2, medida3, maior;

        System.out.println("Digite as tres distancias:");
        medida1 = sc.nextDouble();
        medida2 = sc.nextDouble();
        medida3 = sc.nextDouble();

        if (medida1 > medida2 && medida1 > medida3) {
            maior = medida1;
        }
        else if (medida2 > medida3) {
            maior = medida2;
        }
        else {
            maior = medida3;
        }

        System.out.printf("MAIOR DISTANCIA = %.2f", maior);

        sc.close();
    }
}